h5grove/utils.py

```diff
@@ -126,9 +126,9 @@ def _sanitize_dtype(dtype: np.dtype) -> np.dtype:
 
 
 def sanitize_array(array: Sequence[Number], copy: bool = True) -> np.ndarray:
-    """Ensure array save as .npy can be read back by js-numpy-parser.
+    """Ensure array can be converted to a typedarray.
 
-    See https://github.com/ludwigschubert/js-numpy-parser
+    This is compatible with js-numpy-parser, see https://github.com/ludwigschubert/js-numpy-parser
```

Comment:
This part of the doc can be removed as the previous sentence already explains the purpose.

Reply:
```suggestion
```